### PR TITLE
chore!: hide internal styling props

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** #1145 

## Summary

Omits internal props used for styling.

### CODE REVIEW

File: src/components/calcite-radio/calcite-radio.tsx [hunk 1]
```diff
@@ -19,13 +19,19 @@ export class CalciteRadio {
   /** The disabled state of the radio. */
   @Prop({ reflect: true }) disabled?: boolean = false;
 
-  /** The focused state of the radio. */
+  /**
+   * The focused state of the radio.
+   * @private
+   */
   @Prop({ reflect: true }) focused = false;
```
Comment (driskull) [inline src/components/calcite-radio/calcite-radio.tsx:26]: I think this is gone in latest branch.

File: src/components/calcite-tile-select/calcite-tile-select.tsx [hunk 2]
```diff
@@ -36,7 +36,10 @@ export class CalciteTileSelect {
   /** The disabled state of the tile select. */
   @Prop({ reflect: true }) disabled = false;
 
-  /** The focused state of the tile select. */
+  /**
+   * The focused state of the tile select.
+   * @private
+   */
   @Prop({ reflect: true, mutable: true }) focused = false;
```
Comment (driskull) [inline src/components/calcite-tile-select/calcite-tile-select.tsx:43]: I think this is gone in latest branch.

File: src/components/calcite-tile/calcite-tile.tsx [hunk 3]
```diff
@@ -22,7 +22,10 @@ export class CalciteTile {
   /** The embed mode of the tile.  When true, renders without a border and padding for use by other components. */
   @Prop({ reflect: true }) embed = false;
 
-  /** The focused state of the tile. */
+  /**
+   * The focused state of the tile.
+   * @private
+   */
   @Prop({ reflect: true }) focused = false;
```
Comment (driskull) [inline src/components/calcite-tile/calcite-tile.tsx:29]: I think this is gone in latest branch.

**Review by driskull — APPROVED** [review]
👍

File: src/components/calcite-radio/calcite-radio.tsx (hunk 1 above)
Comment (jcfranco) [reply src/components/calcite-radio/calcite-radio.tsx:26]: This one is still in master. cc @eriklharper

**Pull request merged**